.eslintrc.json

```diff
@@ -20,83 +19,105 @@
   "rules": {
     "import/no-extraneous-dependencies": "off",
     "import/extensions": "off",
-    "no-null/no-null": "error",
-    "sonarjs/cognitive-complexity": ["error", 30],
+    "sonarjs/cognitive-complexity": [
```

Comment:
why are we making changes to this file?

Reply:
I was thinking that we could use `strict` mode in typescript, but that made different type errors appear. Again, apologies for scattering things about.